apps/src/sites/studio/pages/home/_homepage.js

```diff
@@ -64,9 +64,18 @@ function showHomepage() {
             canViewAdvancedTools={homepageData.canViewAdvancedTools}
             isEnglish={isEnglish}
             ncesSchoolId={homepageData.ncesSchoolId}
+            schoolAddress1={homepageData.schoolAddress1}
+            schoolAddress2={homepageData.schoolAddress2}
+            schoolAddress3={homepageData.schoolAddress3}
+            schoolCity={homepageData.schoolCity}
+            schoolState={homepageData.schoolState}
+            schoolZip={homepageData.schoolZip}
             censusQuestion={homepageData.censusQuestion}
             showCensusBanner={homepageData.showCensusBanner}
+            donorBannerName={homepageData.donorBannerName}
             teacherName={homepageData.teacherName}
+            teacherFirstName={homepageData.teacherFirstName}
+            teacherSecondName={homepageData.teacherSecondName}
```

Comment:
Good grief we pass a lot of props through to this component.  It might be cleaner to pass a single object of `teacherDonorBannerProps` down here, defining its shape explicitly in proptypes.